src/syrupy/serializers/base.py

```diff
@@ -133,7 +162,7 @@ def _ensure_snapshot_dir(self, index: int) -> None:
             pass
 
     @abstractmethod
-    def read_snapshot_from_file(
+    def _read_snapshot_from_file(
```

Comment:
mentioned above, I prefer removing the underscore if we want to allow subclasses to override

Reply:
We can do that with double underscore methods, and still use underscore methods for internal class methods that can be overridden